repository/src/main/resources/alfresco/repository.properties

```diff
@@ -978,6 +978,9 @@ rendition.config.initialAndOnError.cronExpression=0/10 * * * * ?
 # files (possibly added to a volume via k8 ConfigMaps).
 rendition.config.dir=shared/classes/alfresco/extension/transform/renditions
 
+# Optional property to enable storage of renditions as node properties

```

Comment:
Add clear comment to mark it as "experimental - unsupported (not for production use)" (or similar). Use same comment through-out to make it easy to find.